src/realm/object-store/sync/app.hpp

```diff
@@ -266,8 +266,9 @@ class App : public std::enable_shared_from_this<App>,
 
     /// Refreshes the custom data for a specified user
     /// @param user The user you want to refresh
+    /// @param refresh_location If true, the location metadata will be updated before refresh
     void refresh_custom_data(const std::shared_ptr<SyncUser>& user,
-                             util::UniqueFunction<void(util::Optional<AppError>)>&&);
+                             util::UniqueFunction<void(util::Optional<AppError>)>&&, bool refresh_location = false);
```

Comment:
Can we make this new boolean not the last parameter and not have a default value? If I'm really being pedantic here I should ask for it to be a TaggedBool or have two different overloads, but I won't be that pedantic.